internal/cli/apps.go

```diff
@@ -86,6 +86,7 @@ var CmdAppShow = &cobra.Command{
 	},
 }
 
+// TODO: When pushing the app again after scaling the instances is reset (?)
```

Comment:
I think that's ok? The user executed the push with that number of instances. 

How do you tell the difference between default value and intentional change?
> epinio push --instances 1
> epinio push
(Well, yes, we could make instances a pointer and if it's nil try to do the right thing)

Reply:
I created a new issue for this: https://github.com/epinio/epinio/issues/403

I don't know what the implementation will look like, probably what you suggested with the pointer. In any case, if there is no values explicitly set for instances, we should maintain the current value.